guides/common/modules/proc_configuring-server-to-synchronize-content-over-a-network.adoc

```diff
@@ -24,31 +32,35 @@ Default is `Library`.
 Default is `Default_Organization_View`.
 . From the *SSL CA Content Credential* menu, select a CA certificate used by the upstream {ProjectServer}.
 . Click *Update*.
-
-The repositories should appear under *Content* > *Products*.
-
-You can synchronize the repositories using the *Sync Now* feature.
-You can also create a synchronization plan to ensure updates on a regular basis. For more information, see {ContentManagementDocURL}Creating_a_Sync_Plan_content-management[Creating a Synchronization Plan] in the _Content Management Guide_.
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click *Sync Now* to synchronize the repositories.
++
+You can also create a synchronization plan to ensure updates on a regular basis.
+For more information, see {ContentManagementDocURL}Creating_a_Sync_Plan_content-management[Creating a Synchronization Plan] in the _Content Management Guide_.
 
 .CLI Procedure
-. Log in to your {ProjectServer} using SSH.
-. Display information about your organization: // TODO upstream or downstream?
+. Connect to your downstream {ProjectServer} using SSH.
+. View information about the upstream CA certificate:
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer organization show --name="_My_Organization_"
+# hammer content-credential show --name="_My_Downstream_Organization_" --name="_upstream_ca_cert_"
```

Comment:
```suggestion
# hammer content-credential show --organization-label="_My_Downstream_Organization_" --name="_upstream_ca_cert_"
```
We need to pick Organization, not necessarily by label, but one of `--organization[-id|-title|-label]`

Reply:
Yes and I'm giving an example using this particular one, because it looks pretty. I'm assuming that the user knows the other options or knows how to display help. Will such example suffice in that case?

Reply:
Allow me to chime in:

```
$ git grep -- "--organization" | wc
     218    1026   26992
$ git grep -- "--organization-" | wc
      10      53    1373
```

I'd argue it's rather unusual to use `--organization-label`. :v:

Reply:
I am completely fine to use `--organization`, I just wanted to emphasize that using 2x `--name` and 0x `--orgaization[whatever]` will not work.

Let me give an example:
```
[root@sat ~]# hammer content-credential show --name="Default Organization" --name="SourceCA"
Error: Missing options to search organization.
[root@sat ~]# hammer content-credential show --organization="Default Organization" --name="SourceCA"
Id:           1
Name:         SourceCA
Organization: Default Organization
Content:      
  -----BEGIN CERTIFICATE-----
  MIIHCjCCBPKgAwIBAgIJAMIVsuCKkypEMA0GCSqGSIb3DQEBCwUAMIGMMQswCQYD
  VQQGEwJVUzEXMBUGA1UECAwOTm9ydGggQ2Fyb2xpbmExEDAOBgNVBAcMB1JhbGVp
  Z2gxEDAOBgNVBAoMB0thdGVsbG8xFDASBgNVBAsMC1NvbWVPcmdVbml0MSowKAYD
  ...
  ZSYOhxGbcsh5eDvDHWUwmTlvwB+f2sHLR1d+QNkp
  -----END CERTIFICATE-----

```

Reply:
Ohhh, I get it now. So if I use -name-, we're cool?